src/SourceCode.Clay/Number.cs

```diff
@@ -929,20 +1018,30 @@ int IConvertible.ToInt32(IFormatProvider provider)
             }
         }
 
-        long IConvertible.ToInt64(IFormatProvider provider)
+        public double ToInt64()
```

Comment:
New method override, optimized

Reply:
Copy the body of ToInt64 and just do a straight cast.